src/sql/workbench/contrib/notebook/browser/notebookActions.ts

```diff
@@ -46,11 +46,12 @@ export class AddCellAction extends Action {
 	public cellType: CellType;
 
 	constructor(
-		id: string, label: string, cssClass: string
+		id: string, label: string, cssClass: string,
+		@INotebookService private _notebookService: INotebookService
 	) {
 		super(id, label, cssClass);
 	}
-	public async run(context: INotebookEditor | CellContext): Promise<void> {
+	public async run(context: URI | CellContext): Promise<void> {
```

Comment:
Note - at some point we may want to do something similar for the CellContext here. But currently as extensions can only contribute actions to the main toolbar this isn't a concern here so I left it as is.